build/__detectorNugetPackagesVersions.sh

```diff
@@ -0,0 +1,5 @@
+# This file was auto-generated from 'constants.yaml'. Changes may be overridden.
+
+VERSION_PREFIX='1.0.0'
+VERSION_SUFFIX="pre-$BUILD_BUILDNUMBER"
```

Comment:
why the `pre-` prefix?

Reply:
Let's say we make a release of a nuget package today with a version `1.0.0`. Let's say we would then update this version to `1.0.1` or later, we want to release them as `prerelease` packages and after our customers have tried and tested them, we can then make a release by remove the `prerelease` prefix